ceph-salt.8

```diff
@@ -0,0 +1,213 @@
+\" Man page for ceph-salt
+.TH "ceph-salt" "8" "" "" "ceph-salt"
+.SH "NAME"
+ceph\-salt \- deploy ceph clusters using cephadm
+.SH "SYNOPSIS"
+.sp
+\fBceph-salt\fP [\fI\-\-global\-opts\fP] \fIcommand\fP
+[\fI\-\-command\-opts\fP] [\fIcommand\-arguments\fP]
+.sp
+\fBceph-salt\fP [\fIcommand\fP] \fB\-\-help\fP
+.sp
+\fBceph-salt\fP \fB\-\-help\fP
+.SH "DESCRIPTION"
+.sp
+ceph\-salt is a command\-line tool for deploying ceph clusters, managed by
+cephadm, using Salt.
+.SH "COMMANDS"
+.sp
+\fBapply\fP [\fIoptions\fP] \fI[minion_id]\fP
+.RS 4
+Apply configuration by running ceph-salt formula in order to provision hosts
+so they can be used as ceph cluster nodes. On first execution, a new ceph
+cluster with a single MON and MGR will be deployed. Apply command is
+idempotent, so it can be executed multiple times.
+.sp
+\fB\-n\fP, \fB\-\-non\-interactive\fP
+.RS 4
+Execute without opening a graphical user interface
+.RE
+.sp
+\fBminion_id\fP
+.RS 4
+The minion that should be configured. If not specified, all ceph-salt minions
+will be configured.
+.RE
+.RE
+.sp
+\fBconfig\fP [\fIpath\fP] \fI[subcommand]\fP
+.RS 4
+Open an interactive shell where you can manipulate ceph-salt's configuration.
+Each configuration option is present under a configuration group. You can
+navigate through the groups and options using the familiar \fBls\fP and
+\fBcd\fP commands similar to a regular shell. In each path you can type
+\fBhelp\fP to see the available commands. Different options might have
+different commands available. If both path and subcommand are provided, the
+subcommand will be directly executed without opening the interactive shell.
+.sp
+\fBpath\fP
+.RS 4
+Configuration path where subcommand should be executed
+.RE
+.sp
+\fBsubcommand\fP
+.RS 4
+Subcommand to execute on configuration path
+.RE
+.RE
+.RE
+.sp
+\fBdisengage-safety\fP
+.RS 4
+Disable safety so dangerous operations like purging the whole cluster can be
+performed
+.RE
+.sp
+\fBexport\fP [\fIoptions\fP]
+.RS 4
+Export configuration
+.sp
+\fB\-p\fP, \fB\-\-pretty\fP
+.RS 4
+Pretty-prints JSON ouput
```

Comment:
Pretty prints JSON output - usnure if there's a particular reason you're using a dash here?

Reply:
"pretty-prints" is OK in U.S. usage

Reply:
:woman_shrugging: cool